frontend/awx/resources/projects/ProjectPage/ProjectPage.tsx

```diff
@@ -50,6 +50,8 @@ export function ProjectPage() {
       { label: t('Access'), page: AwxRoute.ProjectAccess },
       { label: t('Schedules'), page: AwxRoute.ProjectSchedules },
       { label: t('Job templates'), page: AwxRoute.ProjectJobTemplates },
+      { label: t('Users'), page: AwxRoute.ProjectUsers },
```

Comment:
Let's rename the tabs as User Access and Team Access to be consistent with the experience for EDA resources.

Reply:
Fixed.